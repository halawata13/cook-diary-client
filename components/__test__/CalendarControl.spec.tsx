import React from 'react';
import renderer, { act } from 'react-test-renderer';
import CalendarControl from "../CalendarControl";

describe('CalendarControl', () => {
  let component: renderer.ReactTestRenderer;
  let tree: renderer.ReactTestRendererJSON | renderer.ReactTestRendererJSON[] | null;

  it('should match to snapshot', () => {
    const date = new Date(2021, 12, 21);

    act(() => {
      component = renderer.create(<CalendarControl date={date} onDateChanged={() => {}} />);
      tree = component.toJSON();
    });

    expect(tree).toMatchSnapshot();
  });
});
