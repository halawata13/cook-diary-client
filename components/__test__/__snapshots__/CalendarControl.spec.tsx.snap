// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarControl should match to snapshot 1`] = `null`;
